docs/configure-artifact-repository.md

```diff
@@ -107,43 +111,181 @@ $ cat > policy.json <<EOF
    ]
 }
 EOF
-$ aws s3 mb s3://$mybucket [--region xxx]
-$ aws iam create-user --user-name $mybucket-user
-$ aws iam put-user-policy --user-name $mybucket-user --policy-name $mybucket-policy --policy-document file://policy.json
-$ aws iam create-access-key --user-name $mybucket-user > access-key.json
 ```
 
 If you do not have Artifact Garbage Collection configured, you should remove `s3:DeleteObject` from the list of Actions above.
 
-NOTE: if you want argo to figure out which region your buckets belong in, you
-must additionally set the following statement policy. Otherwise, you must
-specify a bucket region in your workflow configuration.
+!!! Note "Discovering bucket regions"
+    If you want Argo Workflows to figure out which region your buckets belong in, you must additionally set the following statement policy.
+    Otherwise, you must specify a bucket region in your workflow configuration.
```

Comment:
```suggestion
!!! Note "Region discovery"
    Argo can discover the region of your buckets with the additional policy below.
    Without this, you must specify the region in your artifact configuration.
```

simplify.

also the below example needs to add 4 space indentation to the entire code-block so that it will show up under the note. It might make sense to change this to a heading instead of a large note?

Reply:
It was originally a really bad 

> NOTE: If you want... 
> `code`

block, so I just wanted to improve it a little bit here. 

I originally tried indenting the code block but it failed the linting rules due to the code block being indented instead of fenced (even though it was both). Happy to change this to a heading.

Reply:
Yea you need to temporarily disable the linting rule to get it to work; can see an example in this review comment & commit: https://github.com/argoproj/argo-workflows/pull/12809#issuecomment-2036237459